articles/index.yml

```diff
@@ -66,10 +66,10 @@ landingContent:
             url: machines/index.md
       - linkListType: quickstart
         links:
-          - text: Build a quantum random number generator
-            url: quickstarts/qrng.md
           - text: Quantum basics with Q#
             url: quickstart.md
+          - text: Build a quantum random number generator
```

Comment:
I'm not sure if we should change the order here. The QRNG tutorial was created because the Bell example that appears in _Quantum basics with Q#_ was a bit complicated for a "quantum hello world" tutorial.

Reply:
In that case, the order in TOC needs to change for consistency.

Reply:
And we need to change the title "Quantum basics with Q#" as it is misleading. Any suggestions of an alternative?

Reply:
Yup, I agree, how about: "Exploring entanglement with Q#"?

Reply:
That's a good title.